src/App.css

```diff
@@ -5,244 +5,335 @@
 
 /* Leaderboard */
 .home-button {
-  font-size: 3rem;
-  float: left;
-}
-
-.mainText-leaderboard {
-  padding-top: 2%;
-  font-family: 'Quicksand', sans-serif;
-  font-weight: bold;
-  font-size: 3em;
-  width: 90%;
-  margin-left: 5%;
+    font-size: 3rem;
+    float: left;
 }
 
 .colNames {
-  font-family: 'Quicksand', sans-serif;
-  border: none !important;
-  padding: 0px !important;
+    font-family: 'Quicksand', sans-serif;
+    border: none !important;
+    padding: 0px !important;
 }
 
 .dataText {
-  font-family: 'Quicksand', sans-serif;
-  border: none !important;
-  padding: 0px !important;
-  max-width: 50%;
+    font-family: 'Quicksand', sans-serif;
+    border: none !important;
+    padding: 0px !important;
+    max-width: 50%;
 }
 
 .tablecss {
-  border-spacing: 0;
-  border-collapse: collapse;
-  font-size: 1.5em;
-  text-align: center;  
+    border-spacing: 0;
+    border-collapse: collapse;
+    font-size: 1.5em;
+    text-align: center;
 }
 
 .tableContainer {
-  margin-top: 2%;
-  margin-left: 10%;
-  margin-right: 10%;
-  border: 4px solid black;
-  border-radius: 15px;
-  max-width: 90%;
+    margin-top: 2%;
+    margin-left: 10%;
+    margin-right: 10%;
+    border: 4px solid black;
+    border-radius: 15px;
+    max-width: 90%;
 }
 
 .leaderboard-div {
-  text-align: center;
-  height: 100vh;
-  min-height: 100vh;
-  background-color: #8FF4F4;
+    text-align: center;
+    height: 100vh;
+    min-height: 100vh;
+    background-color: #8FF4F4;
 }
 
 /* Upgrades */
 .upgrades-div {
-  text-align: center;
-  height: 100vh;
-  min-height: 100vh;
-  background-color: #8FF4F4;
+    text-align: center;
+    height: 100vh;
+    min-height: 100vh;
+    background-color: #8FF4F4;
 }
 
-.mainText-upgrades {
-  padding-top: 2%;
-  font-family: 'Quicksand', sans-serif;
-  font-weight: bold;
-  font-size: 3em;
-  width: 90%;
-  margin-left: 5%;
+.mainTitle {
+    padding-top: 2%;
+    font-family: 'Quicksand', sans-serif;
+    font-weight: bold;
+    font-size: 3em;
+    width: 90%;
+    margin-left: 5%;
 }
 
 /* Homepage */
 .homePage {
-  background-color: #8FF4F4;
-  height: 100vh;
-  min-height: 100vh;
+    background-color: #8FF4F4;
+    background-image: url("https://cdn.discordapp.com/attachments/667630015554256896/814979438462238740/6480375.jpg");
```

Comment:
I would recommend retrieving the background-image from a local file stored inside our project repo, instead of retrieving it online. We are not sure whether our image on the Discord server are still available in the long term. 
One possible approach is adding the image file inside the src folder, and have the URL links to that file.